proposals/custom-descriptors/Overview.md

```diff
@@ -536,7 +536,115 @@ console.log(counter.get()); // 1
 
 ## Declarative Prototype Initialization
 
-TODO
+We expect toolchains to need to configure thousands of JS prototypes and tens of thousands of methods,
+so to reduce startup latency we want a declarative method of constructing, importing,
+and populating the `DescriptorOptions` objects.
```

Comment:
Could there be more of a separation between the statement of requirements ("need the configure thousands of ...") and the proposed technical solution (the declarative custom section) which indicates that there are other potential technical solutions to explore and weigh (such as extending JS built-ins and the expressivity of global initializer expressions), and what follows is what is currently being explored/prototyped?  I'd like to avoid closing the door on alternative approaches until we have a better understanding (which we'll get from prototyping the declarative custom section, esp. if there's a polyfill, so it's great that we're prototyping).